app/models/spree/gateway/stripe_sca.rb

```diff
@@ -86,6 +124,16 @@ def ensure_enterprise_selected
 
         errors.add(:stripe_account_owner, I18n.t(:error_required))
       end
+
+      def full_checkout_path
+        URI.join(url_helpers.root_url, url_helpers.checkout_path).to_s
+      end
+
+      def url_helpers
+        # This is how we can get the helpers with a usable root_url outside the controllers
```

Comment:
Do you think this would be useful elsewhere in the app?

Reply:
yes, for sure. I cant find an example so I decided not to extract it yet...